vibrato/src/dictionary/lexicon/map/trie.rs

```diff
@@ -55,20 +55,20 @@ impl Trie {
             .common_prefix_search(input.iter().cloned())
             .map(move |(value, end_char)| {
                 // Safety: input.len() is no more than 0xFFFF.
-                TrieMatch::new(value, unsafe { u16::try_from(end_char).unwrap_unchecked() })
+                TrieMatch::new(value, end_char)
```

Comment:
Could you remove the above comment for Safety?

Reply:
fixed